packages/netlify-cms-backend-gitlab/src/API.ts

```diff
@@ -291,65 +346,37 @@ export default class API {
     flow([
       unsentRequest.withMethod('GET'),
       this.request,
-      p => Promise.all([p.then(this.getCursor), p.then(this.responseToJSON)]),
+      p =>
+        Promise.all([
+          p.then(this.getCursor),
+          p.then(this.responseToJSON).catch((e: FetchError) => {
+            if (e.status === 404) {
+              return [];
+            } else {
+              throw e;
+            }
+          }),
+        ]),
       then(([cursor, entries]: [Cursor, {}[]]) => ({ cursor, entries })),
     ])(req);
 
-  reversableActions = Map({
```

Comment:
GitHub returns by path ascending by default, Bitbucket was returning by path descending (but was meant to return by path ascending based on code comments) and GitLab was returning by path ascending, but we were reversing pagination links to show by path descending.
Aligned all backends to return by path ascending.